docs/api/config/js_kanban_columnshape_config.md

```diff
@@ -34,7 +34,8 @@ columnShape?: {
     }),
     collapsedTemplate?: template(column => {
         return "The HTML template of the column header in the collapsed state";
-    })
+    }),
+    confirmDeletion?: boolean | { show?: boolean }
```

Comment:
see my previous comment please: all other shapes have this setting as boolean only